packages/logs/src/boot/startLogs.spec.ts

```diff
@@ -170,6 +170,31 @@ describe('logs', () => {
       expect(server.requests.length).toEqual(0)
       expect(sendSpy).toHaveBeenCalled()
     })
+
+    it('should send console logs if ff "forward-logs" enabled', () => {
+      const errorLogger = new Logger(noop)
+      const logErrorSpy = spyOn(errorLogger, 'log')
+
+      updateExperimentalFeatures(['forward-logs'])
+      startLogs({ errorLogger })
+
+      /* eslint-disable no-console */
+      console.log('foo', 'bar')
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**❗ warning:** ‏wrong eslint syntax (same below)
```suggestion
      // eslint-disable-next-line no-console
      console.log('foo', 'bar')
```